system_setup/cmd/server.py

```diff
@@ -74,6 +79,21 @@ def main():
     logger.info("Starting System Setup server revision {}".format(version))
     logger.info("Arguments passed: {}".format(sys.argv))
 
+    prefillFile = opts.prefill
+    if prefillFile:
```

Comment:
I'm not going to hold the merge for this as it's more a matter of preference, but is there any reason to scan for the file here and actually parse it later?  This could be in a catch block in system_setup/server/server.py if you want this info for diagnostic purposes.

Reply:
Matter of preference, as you said. My reasoning was delaying parsing to the closest point of usage, but doing server side sanity checks the earliest possible to avoid trying to parse an invalid file. Maybe my checks were too shallow, but that was the intent.